bot.py

```diff
@@ -86,6 +86,22 @@ async def soon(ctx, player=None):
 
         await sendGames(ctx, games, (player == None))
 
+    @bot.command(
+        help=bot.command_prefix + "next <?name?> - Shows next game for the person requested"
+    )
+    async def next(ctx, player=None):
+        games = retrieveAllGames(teams, player)
+        nextEmbed = DegenEmbed(title=f"Next Game for {player}", description=None, color=discord.Color.red())
+        nextEmbed.create("https://avatars.githubusercontent.com/u/1737241?v=4")
+
+        time_now = datetime.now()
+        for game in games:
+            if game.gametime >= time_now:
+                next_game = [game]
```

Comment:
what's this variable for?

Reply:
The way you send the game objects requires it to be a list.

Reply:
Which could probably use some refactoring down the road but that's bigger than this command. So I had to take the game and make is a list, otherwise is was growing an error, effectively saying next_game does not have the field len() or something.